dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/AbstractJdbcEventAnalyticsManager.java

```diff
@@ -232,7 +232,19 @@ else if ( !params.hasPeriods() && params.hasFilterPeriods() )
                 ProgramIndicator in = (ProgramIndicator) queryItem.getItem();
 
                 String asClause = " as " + quote( in.getUid() );
-                columns.add( "(" + programIndicatorService.getAnalyticsSql( in.getExpression(), in, params.getEarliestStartDate(), params.getLatestEndDate() ) + ")" + asClause );
```

Comment:
The AbstractJdbcEnrollmentAnalyticsManager overrides the getWhereClause method, and also has a call to programIndicatorService.getAnalyticsSql - I would think you need to add a similar join to that method to make the enrollment analytics work?